packages/mdc-segmented-button/segment/_segment.scss

```diff
@@ -40,30 +54,65 @@
       }
     }
 
+    // Removes ripple's focus outline
+    &:focus {
+      @include feature-targeting.targets($feat-structure) {
+        outline-width: 0;
+      }
+    }
+
     &:first-child {
       @include feature-targeting.targets($feat-structure) {
-        border-radius: segment-theme.$border-radius 0px 0px
+        border-radius: segment-theme.$border-radius 0 0
           segment-theme.$border-radius;
       }
     }
 
     &:last-child {
       @include feature-targeting.targets($feat-structure) {
         border-right-width: segment-theme.$border-width;
-        border-radius: 0px segment-theme.$border-radius
-          segment-theme.$border-radius 0px;
+        border-radius: 0 segment-theme.$border-radius
+          segment-theme.$border-radius 0;
       }
     }
+
+    .mdc-segmented-button__touch {
+      @include touch-target-mixins.touch-target($query: $query);
+    }
+
+    .mdc-segmented-button--touch {
+      @include touch-target-mixins.margin(
+        $component-height: 32px,
+        $query: $query
+      );
+    }
```

Comment:
These should probably be renamed to `.mdc-segmented-button__segment__touch` and `.mdc-segmented-button__segment--touch`.

Reply:
`.mdc-segmented-button__segment--touch` (Consistent with `mdc-button--touch`)